src/java/htsjdk/variant/variantcontext/GenotypesContext.java

```diff
@@ -39,7 +40,9 @@
 /**
  * Represents an ordered collection of Genotype objects
  */
-public class GenotypesContext implements List<Genotype> {
+public class GenotypesContext implements List<Genotype>, Serializable {
```

Comment:
`unparsedGenotypeData` is serializable in practice, since it's always of type `String` or `byte[]`